README.md

```diff
@@ -837,6 +837,137 @@ OPTIONS:
 --timestamps, -t           [Optional] Shows timestamps on each line in the log output.
 ```
 
+## Using Private Registry Authentication
+
+If you want to use privately hosted container images with ECS, the ECS CLI can store your private registry credentials for you with AWS Secrets Manager and create an IAM role which ECS can use to access the credentials and private images. This allows you to:
```

Comment:
suggestion: "the ECS CLI can store your private registry credentials in AWS Secrets Manager"